src/tests/sphmultipcqueue_t.c

```diff
@@ -888,6 +906,12 @@ main(int argc, char *argv[]) {
 	int test_id,i;
 	int num_producers, num_consumers;
 
+	for (i = 1; i < argc; i++) {
+		cpu_list[i-1] = strtol(argv[i],0,0);
+		if (cpu_list[i-1] > cpu_max) cpu_max = cpu_list[i-1];
```

Comment:
ok. can we also print success/failure?
if (rc == 0)
 +    printf("%6d: sched_setaffinity(thread %d to CPU %d)=%d\n",
 +      sphFastGetTID(),tn,cpu_list[tn],rc);
else 
  +    printf("%6d: sched_setaffinity(thread %d to CPU %d) failed with rc %d\n",
 +      sphFastGetTID(),tn,cpu_list[tn],rc);